extensions/interactions/NumberWithUnits/directives/number_with_units_help_modal_directive.html

```diff
@@ -0,0 +1,149 @@
+<div class="modal-header">
+  <b>Guidelines for writing units</b>
+</div>
+<div class="modal-body">
+  <style>
+    td {
+      padding: 5px;
+      width: 30%;
+    }
+  </style>
+  <table class="table-bordered">
+    <tr>
+      <td><b>Rule Description</b></td>
+      <td><b>Incorrect Format</b></td>
+      <td><b>Correct Format</b></td>
+    </tr>
+    <tr>
+      <td>
+        Separate value and the unit with one space.
+      </td>
+      <td>3km</td>
+      <td>3 km</td>
+    </tr>
+    <tr>
+      <td>
+        In case the value is a fraction, separate numerator and denominator by
```

Comment:
`If the value is a fraction,...by a forward slash...`

Reply:
Done.